db/db_impl/db_impl_compaction_flush.cc

```diff
@@ -848,11 +848,19 @@ Status DBImpl::CompactRange(const CompactRangeOptions& options,
   if (options.change_level) {
     ROCKS_LOG_INFO(immutable_db_options_.info_log,
                    "[RefitLevel] waiting for background threads to stop");
+    DisableManualCompaction();
     s = PauseBackgroundWork();
+    bool bg_paused = false;
```

Comment:
Minor but we could achieve the same effect by moving `ContinueBackgroundWork` inside the `if` block, right?

Reply:
Yup, done.